src/cli/cli_srp_client.cpp

```diff
@@ -327,6 +638,21 @@ template <> otError SrpClient::Process<Cmd("service")>(Arg aArgs[])
         }
     }
 #if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+    /**
+     * @cli srp client service key (set, get)
+     * @code
+     * srp client service key enable
+     * Done
+     * @endcode
+     * @code
+     * srp client service key
+     * Enabled
+     * Done
+     * @endcode
+     * @par
+     * Gets or sets the service key record inclusion mode in the SRP client.
```

Comment:
Maybe useful to add comment here that mentions that this is intended for testing only.

Reply:
And that it is available under `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE`.